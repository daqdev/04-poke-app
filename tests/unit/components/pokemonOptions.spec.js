import { shallowMount } from "@vue/test-utils";
import PokemonOptions from '@/components/PokemonOptions'
import { pokemonzMok } from "../mocks/pokemons.mock";

describe('pokemon-options', ()=>{

    let wrapper
    beforeEach(()=>{

        wrapper = shallowMount(PokemonOptions, {
            props:{
                pokemons:pokemonzMok
            }
        })

    })

    test('que haga match con el snapshot', ()=>{

        // console.log(pokemonzMok)
        // console.log(wrapper.html())
        
        expect(wrapper.html()).toMatchSnapshot()
    })
})