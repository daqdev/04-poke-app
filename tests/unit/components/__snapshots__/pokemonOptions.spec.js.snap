// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pokemon-options que haga match con el snapshot 1`] = `
<div class="options-container">
  <ul>
    <li>bulbasaur</li>
    <li>ivysaur</li>
    <li>venusaur</li>
    <li>charmander</li>
  </ul>
</div>
`;
